core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -351,7 +351,16 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
   def getAppId: String = get("spark.app.id")
 
   /** Does the configuration contain a given parameter? */
-  def contains(key: String): Boolean = settings.containsKey(key)
+  def contains(key: String): Boolean = {
+    if (settings.containsKey(key)) {
```

Comment:
It always bothers me when I see `if (true) true`. I think `settings.containsKey(key) || ...`